mapa.lua

```diff
@@ -80,6 +80,10 @@ local function setCharacter( )
   -- no próprio tiled
   character = map:findObject("character")
 
+  local start = map:findObject("start")
+  character.x = start.x 

```

Comment:
Estou tendo erro nessa linha do código ao tentar simular o jogo.

15:11:21.451  ERROR: Runtime error
15:11:21.451  C:\Users\gnsod\Downloads\RatinhoExplorador-minigameMapa\mapa.lua:84: attempt to index local 'start' (a boolean value)
15:11:21.451  stack traceback:
15:11:21.451  	C:\Users\gnsod\Downloads\RatinhoExplorador-minigameMapa\mapa.lua:84: in function 'setCharacter'
15:11:21.451  	C:\Users\gnsod\Downloads\RatinhoExplorador-minigameMapa\mapa.lua:484: in function '?'
15:11:21.451  	?: in function 'dispatchEvent'
15:11:21.451  	?: in function 'gotoScene'
15:11:21.451  	C:\Users\gnsod\Downloads\RatinhoExplorador-minigameMapa\main.lua:21: in main chunk